### ISSUE TITLE
Polar plot title and axes have odd text-shadow that cannot be changed?

### ISSUE THREAD

**Issue by djkashtan** [plotly/plotly.js#97]
If you look at the css or just simply mouse-highlight the title or axes on a polar plot, the text has a white text-shadow. I could not find an option to change it. The legend does not have a text shadow and other plot titles and axes do not appear to have text-shadows by default. Can anybody help me? My website does not have a white background and my polar plot looks really ugly with the white text-shadow.

This link has polar plots that demonstrate the behavior: https://plot.ly/javascript/polar-chart/

**Comment by etpinard** [comment]
From v1.0.0 release [notes](https://github.com/plotly/plotly.js/releases/tag/v1.0.0): 

> Polar charts are under-developed.

**Comment by emackey** [comment]
This is already available in the `layout` object of the polar plot.

```
layout: {
    font: {
        outlineColor: 'transparent'  // or repeat layout.paper_bgcolor here
    }
    //...
}
```

**Comment by etpinard** [comment]
fixed in `scatterpolar` after #2200 

see also https://github.com/plotly/plotly.js/issues/1597
